### PR TITLE
Adding configuration properties for JWT audience configuration via UI feature

### PR DESCRIPTION
**Pull request by emswbandara**
part of https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/783
Added the table creation for identity core feature SQL files, and the property updates to identity.xml

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql [hunk 1]
```diff
@@ -1034,3 +1034,24 @@ CREATE TRIGGER IDN_SAML2_ASSERTION_STORE_TRIG NO CASCADE
...
+/
+CREATE TRIGGER IDP_OIDC_PROPERTY_TRIG NO CASCADE
+            BEFORE INSERT
+            ON IDP_OIDC_PROPERTY
+            REFERENCING NEW AS NEW
+            FOR EACH ROW MODE DB2SQL
+                BEGIN ATOMIC
+                    SET (NEW.ID) = (NEXTVAL FOR IDP_OIDC_PROPERTY_SEQ);
+                END
+/
```
Comment (pulasthi7) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql]: Add new line to the end of file

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql [hunk 2]
```diff
@@ -668,3 +668,12 @@ CREATE TABLE IF NOT EXISTS  IDN_SAML2_ASSERTION_STORE (
...
 
+CREATE TABLE IF NOT EXISTS  IDP_OIDC_PROPERTY (
+  ID INTEGER NOT NULL AUTO_INCREMENT,
+  TENANT_ID  INTEGER,
+  CONSUMER_KEY  VARCHAR(255) ,
+  PROPERTY_KEY  VARCHAR(255) NOT NULL,
+  PROPERTY_VALUE  VARCHAR(2047) ,
+  PRIMARY KEY (ID),
+  FOREIGN KEY (CONSUMER_KEY) REFERENCES IDN_OAUTH_CONSUMER_APPS(CONSUMER_KEY) ON DELETE CASCADE
+);
```
Comment (pulasthi7) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql]: Add new line to the end of file, do the same at other places

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql (hunk 1 above)
Comment (emswbandara) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql]: Fixed

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql (hunk 2 above)
Comment (emswbandara) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql]: Fixed

**Pull request merged**